src/main/java/org/jabref/model/entry/Author.java

```diff
@@ -11,13 +13,15 @@
  * <p>
  * Current usage: only methods <code>getLastOnly</code>, <code>getFirstLast</code>, and <code>getLastFirst</code> are used; all other methods are provided for completeness.
  */
+@AllowedToUseLogic("because it needs to use formatter")
 public class Author {
 
     /**
      * Object indicating the <code>others</code> author. This is a BibTeX feature mostly rendered in "et al." in LaTeX.
      * Example: <code>authors = {Oliver Kopp and others}</code>. This is then appearing as "Oliver Kopp et al.".
      * In the context of BibTeX key generation, this is "Kopp+" (<code>+</code> for "et al.") and not "KO".
      */
+    public static final RemoveWordEnclosingAndOuterEnclosingBracesFormatter FORMATTER = new RemoveWordEnclosingAndOuterEnclosingBracesFormatter();
```

Comment:
Very wrong position of the variable. See the javadoc above. This DOES NOT match the new variable, but the `OTHERS` variable.

Please move the variable two lines down (and separate the next one by an empty line, too)